src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  if(days<3){
+    return days*RENT_COST;
+  } else if(days < 7){
+    return days*RENT_COST-20;
+  }
+  return days*RENT_COST-40;
```

Comment:
The function name `calculateRentalCost` is descriptive, which is good. However, the implementation uses hard-coded literals for the discount values and the thresholds for the number of days. Consider using constants with descriptive names to make the code more readable and maintainable.